### DISCUSSION TITLE
disabled plugins

### DISCUSSION THREAD

**Discussion by Dr-wgy** [apache/skywalking#7771] [Q&A] [answered]
If we have this config 
`plugin.exclude_plugins `|Exclude some plugins define in plugins dir.Plugin names is defined in [Agent plugin list](Plugin-list.md)|`""`|
configurations.md

We needn' to delete or remove the specific libraries / jars in `skywalking-agent/plugins/*.jar`
How-to-disable-plugin.md

**Comment by wu-sheng** [comment, marked as answer]
There are 2 ways, yes.
